### PR TITLE
Change page model to handle editing of Ask entries

### PR DESCRIPTION
**Pull request by higs4281**
This is the first step in migrating Ask content to a simpler editing
structure. It exposes content on the AnswerPage and allows version
comparisons.

Metadata will be handled in subsequent steps.

Notes
* The 'tags' and 'tags_es' properties on the Answer model, which are
used in indexing, are being changed to 'clean_tags' and 'clean_tags_es'
to avoid a future collision with the 'tags' field that exists on all Wagtail pages.
* I turned off exports of the content fields (question, statement, snippet, answer) from the Answer model. so that page content won't get clobbered by any changes to an Answer object.
* I added helper text to the AnswerPage's search_tags field to make clear that it's not being used yet.

### CODE REVIEW

File: cfgov/ask_cfpb/models/django.py [hunk 1]
```diff
@@ -547,6 +546,10 @@ def has_live_page(self):
                 return True
         return False
 
+    # def sync_related_questions(self, page):
```
Comment (richaagarwal) [inline cfgov/ask_cfpb/models/django.py]: Can this commented out code be deleted for now?

File: cfgov/ask_cfpb/models/pages.py [hunk 2]
```diff
@@ -523,13 +528,23 @@ class AnswerPage(CFGOVPage):
...
-    snippet = RichTextField(
-        blank=True, help_text='Optional answer intro', editable=False)
-    created_at = models.DateTimeField(auto_now_add=True)
-    updated_at = models.DateTimeField(auto_now=True)
-    publish_date = models.DateTimeField(default=timezone.now)
+    last_edited = models.DateField(
+        blank=True,
+        null=True,
+        help_text="Change the date to today if you make a significant change.")
+    question = models.TextField(blank=True)
```
Comment (richaagarwal) [inline cfgov/ask_cfpb/models/pages.py]: Is there a reason the `question` field becomes a `TextField` in place of a `RichTextField`?

Comment (higs4281) [reply cfgov/ask_cfpb/models/pages.py]: The question has always been a TextField in the Answer model. We didn't see a need to allow  markup in questions.

File: cfgov/ask_cfpb/models/django.py (hunk 1 above)
Comment (higs4281) [reply cfgov/ask_cfpb/models/django.py]: Gone!

File: cfgov/ask_cfpb/models/django.py [hunk 3]
```diff
@@ -556,14 +555,20 @@ def create_or_update_page(self, language=None):
             _parent = english_parent
             _slug = self.slug
             _question = self.question
-            _snippet = self.snippet
-            _answer = self.answer
+            # _snippet = self.snippet
```
Comment (richaagarwal) [inline cfgov/ask_cfpb/models/django.py]: Do you want to remove these or leave them commented out?  In the next iteration we'll be removing this code entirely so it doesn't matter too much either way but might be easier to remove.

File: cfgov/ask_cfpb/models/pages.py [hunk 4]
```diff
@@ -4,19 +4,20 @@
 import re
 from six.moves.urllib.parse import urlparse
 
+# from django import forms
```
Comment (richaagarwal) [inline cfgov/ask_cfpb/models/pages.py]: Can this line be deleted?

Comment (higs4281) [reply cfgov/ask_cfpb/models/pages.py]: good catch. gone!

**Pull request merged**